styles/blocks/01-title.json

```diff
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 3,
+	"title": "Title",
+	"slug": "text-title",
+	"blockTypes": ["core/heading", "core/paragraph"],
```

Comment:
I strongly disagree that a paragraph can be a "title". A paragraph should never look like a heading without being an actual heading, a heading needs to be semantic, it must use the correct HTML element.

Reply:
That is why I preferred the name agreed on in the other pull request, "Lead".

Reply:
It's a very common text style across many editing applications. I've never actually seen "Lead" used. We should not be recreating ideas and instead lead into the expectations users have, already set by many many other applications.

Reply:
What other applications do is not relevant **unless they produce accessible content**, and applying a text style to a paragraph to make it look like a heading means it will be perceived as a heading by sighted users, but not by search engines and screen reader users. Which is not the experience that users expects when using a default theme.

Reply:
I am not understanding the opposition to meet expectations. 

This is not about making headings designed like paragraphs, or paragraphs designed like headings. But rather to provide a set of typographic styles to be applied throughout, wherever necessary—whether that's with making a heading (any level) bigger, or paragraph smaller—it doesn't matter.

This is no different than making a paragraph have XL font size, and a line height of 1.2 (in trunk).

Reply:
I am not understanding the opposition to meet expectations of users that wants or has to create accessible content, and their visitors. 

The difference is that there is a big button at the top of the sidebar that encourages users to turn a paragraph into a heading, while not fulfilling that promise all the way. By only styling the text and not changing the HTML tag.
This is **encouraging** the wrong practice, compared to a text size option which does not encourage but also does not **prevent** wrong practices.

Like I said, the word title and heading are translated into the same word, this already happens in the WordPress interface and editor today. Only using "title" as a style name and replacing "heading" resolves the duplication, but not the whole problem.

Reply:
> The difference is that there is a big button at the top of the sidebar that encourages users to turn a paragraph into a heading, while not fulfilling that promise all the way.

You missed my comments + the commits. I have changed it from "Heading". I had originally asked for feedback on what to name this, to not confuse it with headings. Did you see the commits?

Reply:
I did not miss the commit, I and I wrote:

The words title and heading are translated into the same word, this already happens in the WordPress interface and editor today. Only using "title" as a style name and replacing "heading" resolves the duplication, but not the whole problem.